sdk/tables/azure-data-tables/tests/_shared/testcase.py

```diff
@@ -106,6 +114,36 @@ def create_resource(self, name, **kwargs):
             'storage_account_cs': TableTestCase._STORAGE_CONNECTION_STRING,
         }
 
+
+class GlobalCosmosAccountPreparer(AzureMgmtPreparer):
```

Comment:
As an outside observer it's a little weird to me that you have cosmos_testcase, but in test.case.py you have your globalcosmosaccountpreparer.  From what little I know about storage I _assume_ that's because this operates at some storage-wide level and isn't cosmos-specific?  but I'm admittedly guessing, so take this comment as "I would have been surprised to find this here/to find these preparers organized this way."

The solution may be reoganization, some informative comments, or otherwise, feel free to take the path of least resistance to providing some sort of clarity here.

Reply:
With the new preparer in the devtools this is no longer necessary and I will remove it entirely. Thanks for catching this.